move_group/src/default_capabilities/execute_action_capability.cpp

```diff
@@ -0,0 +1,137 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2016, Willow Garage, Inc.
```

Comment:
i think you could change the company name to your university or something. WG is not writing code in 2016 ;-)